src/executor_utils.c

```diff
@@ -39,6 +41,10 @@ t_err	ft_check_cmd_access(char **args, char **cmd_paths, bool empty_path)
 {
 	t_err	err;
 
+	if (ft_check_dir(args) == ERR_DIR)
```

Comment:
If catch return with err you don't have to call the function twice.

Reply:
see above, will be considered later on